permissions/plugin-mina-sshd-api.yml

```diff
@@ -2,7 +2,7 @@
 name: "mina-sshd-api"
 github: &GH "jenkinsci/mina-sshd-api-plugin"
 paths:
-- "org/jenkins-ci/plugins/mina-sshd-api/mina-sshd-api-parent"
+- "io/jenkins/plugins/mina-sshd-api/mina-sshd-*"
```

Comment:
This has undesirable side effects, like preventing issue tracker links from showing. I recommend not using this hack.

Reply:
I took [plugin-aws-java-sdk.yml](https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/plugin-aws-java-sdk.yml#L8) as an example. @daniel-beck what should be added instead then ?

Reply:
There are also unlikely to be any issue reports associated with this library plugin per se.

Reply:
the CD release is failing with 403 forbidden: https://github.com/jenkinsci/mina-sshd-api-plugin/runs/6873477521?check_suite_focus=true, is that related ?
At first, the `Mina SSHD API :: Parent` succeeded and the next module failed with 403. when restarting the release workflow, now even the parent fails with 403. Maven secrets are present.

Reply:
The parent will fail because we don’t allow overwrite on releases and it got uploaded the first time